tests/torch2/fx/test_compress_weights.py

```diff
@@ -314,6 +316,24 @@ def test_model_devices_and_precisions(use_cuda, dtype):
     assert result.dtype == dtype
 
 
+@pytest.mark.parametrize(
+    "metatype,weight_port_id,ndims,expected_reduction_axes",
+    [
+        (om.PTAtenEmbeddingMetatype, 0, 4, [1]),
+        (om.PTLinearMetatype, 0, 4, [3]),
+        (om.PTMatMulMetatype, 0, 4, [3]),
+        (om.PTMatMulMetatype, 1, 4, [2]),
+        (om.PTAddmmMetatype, 1, 4, [3]),
+        (om.PTAddmmMetatype, 2, 4, [2]),
+        (om.PTMatMulMetatype, 1, 1, [0]),
+        (om.PTMatMulMetatype, 1, 2, [0]),
+    ],
+)
+def test_get_reduction_axes(metatype, weight_port_id, ndims, expected_reduction_axes):
```

Comment:
Test should be defined in tests/torch/test_model_graph_manager.py as other tests for functions from src/nncf/torch/model_graph_manager.py
Test contain only part of metatypes that supports by get_weight_compression_reduction_axes